CODEOWNERS

```diff
@@ -114,10 +114,19 @@ ttnn/ @eyonland @arakhmati @cfjchu @xanderchin @TT-BrianLiu @ayerofieiev-tt @dma
 ttnn/**/kernels/ # Removes the owners above from owning kernels unless specified afterwards
 ttnn/setup.py @tt-rkim
 ttnn/CMakeLists.txt @tt-rkim @ayerofieiev-tt
+ttnn/cpp/ttnn/operations/ccl/ @SeanNijjar @TT-BrianLiu
+ttnn/cpp/ttnn/operations/reduce_scatter/ @SeanNijjar @TT-BrianLiu
+ttnn/cpp/ttnn/operations/pool/ @mywoodstock @nsmithtt
+ttnn/cpp/ttnn/operations/upsample/ @mywoodstock @nsmithtt
```

Comment:
please replace @nsmithtt with @shwetankTT

Reply:
I did add @pavlejosipovic.   I'd like to start with Tenstorrent employees and see how it goes.

Reply:
@shwetankTT  and I are Tenstorrent employees.